alephium/caip10.md

```diff
@@ -0,0 +1,71 @@
+---
+namespace-identifier: alephium-caip10
+title: Alephium Namespace - Addresses
+author: Hongchao Liu (@h0ngcha0)
+discussions-to: https://github.com/ChainAgnostic/namespaces/pull/xxxx
+status: Draft
+type: Standard
+created: 2024-06-19
+updated: 2024-06-19
+requires: ["CAIP-2", "CAIP-10"]
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
```

Comment:
It seems like the `CAIP-10` link is missing here.

Reply:
Seems that the link seems to have worked for other chains, e.g. [algorand](https://namespaces.chainagnostic.org/algorand/caip10).

Reply:
I see the issue now, will add the link at the bottom.